lib/validation/validator.js

```diff
@@ -70,18 +93,29 @@ let validator;
  * @param {string} options.yaml.path Path of the YAML file
  * @param {string} options.yaml.source Content of the YAML file
  * @param {number} [options.yaml.documentIndex=0] Document index in case the YAML file contains multiple documents
+ * @param {SCHEMA_VARIANTS} options.schemaName The name of the schema to validate
  * @throws {@ui5/project/validation/ValidationError}
  *   Rejects with a {@link @ui5/project/validation/ValidationError ValidationError}
  *   when the validation fails.
  * @returns {Promise<undefined>} Returns a Promise that resolves when the validation succeeds
  */
 export async function validate(options) {
-	if (!validator) {
+	const schemaName = options.schemaName;
+
+	if (!SCHEMA_VARIANTS[schemaName]) {
```

Comment:
This check is already done in the Validator constructor. Maybe we don't need both?

Since the other check is covered by tests while this one [isn't](https://coveralls.io/builds/56007649/source?filename=lib%2Fvalidation%2Fvalidator.js#L106), maybe we can remove it?